OSDOCS-9612: Added instructions for the additional security groups tag for ROSA with HCP clusters (#72838)
openshift/openshift-docs: modules/creating-a-machine-pool-ocm.adoc

ifdef::openshift-rosa,openshift-dedicated[]
. Optional: Select additional custom security groups to use for nodes in this machine pool. You must have already created the security groups and associated them with the VPC that you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for security groups in the "Additional resources" section.
+
[IMPORTANT]
====

> You can only use ten additional security groups for machine pools on {hcp-title} clusters. 

Reviewer: Same comments as above

> You can only use, at most, ten additional security groups for machine pools on {hcp-title} clusters. 

Reviewer 2: ```suggestion
You can use up to ten additional security groups for machine pools on {hcp-title} clusters. 
```
